devicon.json

```diff
@@ -7814,6 +7814,31 @@
         "color": "#173aee",
         "aliases": []
     },
+    {
+        "name": "scikitlearn",
+        "altnames": [
+            "scikit-learn",
+            "scikit",
+            "sklearn"
+        ],
+        "tags": [
+            "machine-learning",
+            "python"
```

Comment:
I also think that this will be very small for a square viewbox. Moreover, I am still not clear on which version is which. Although [this page](https://github.com/devicons/devicon/wiki/SVG-Versions) is useful for most cases, there are cases where it's not as clear what to choose.

Initially, I thought that the logic is "if it has letters is a `-wordmark,`" but there are some cases (e.g., AWS, scikitlearn) where there isn't a version without the letters. What would help is to add a flowchart to the page on how to choose the version. 

@Snailedlt Let me know if you need me to rename any files here, and I will do so.

Reply:
@ConX Agreed, it's not always clear which version to choose. 

In addition to the rules in the [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) wiki page the rules for wordmark icons is like this:
1. If the icon contains text, then it should have the `wordmark` suffix.
    - If there is no "non-wordmark" version, and the icon is used in the font, there should be an alias between the `wordmark` and the `non-wordmark` versions like so:
```json
        "aliases": [
          {
            "base": "plain-wordmark",
            "alias": "plain"
          },
        ]
```

Reply:
Though I'm not sure why we only have aliases for fonts. Maybe we can just add them to all versions to make it easier?